<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>4</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Service name</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Username</string>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Addition date</string>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Group</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPasswords">
    <property name="title">
     <string>Management</string>
    </property>
    <addaction name="actionAddPassword"/>
    <addaction name="actionEditPassword"/>
    <addaction name="actionDeletePassword"/>
    <addaction name="separator"/>
    <addaction name="actionExportPassword"/>
    <addaction name="actionDeleteAllPasswords"/>
   </widget>
   <addaction name="menuPasswords"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddPassword">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>Add Password</string>
   </property>
   <property name="toolTip">
    <string>Add a new password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDeletePassword">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditDelete"/>
   </property>
   <property name="text">
    <string>Delete Password</string>
   </property>
   <property name="toolTip">
    <string>Delete selected password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEditPassword">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditPaste"/>
   </property>
   <property name="text">
    <string>Edit Password</string>
   </property>
   <property name="toolTip">
    <string>Edit selected password</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExportPassword">
   <property name="text">
    <string>Export Password...</string>
   </property>
   <property name="toolTip">
    <string>Select passwords and export to preset formats</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDeleteAllPasswords">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>Delete All Passwords</string>
   </property>
   <property name="toolTip">
    <string>Delete all password from the manager</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
